plugins/scaffolder/src/components/TemplateEditorPage/DryRunContext.tsx

```diff
@@ -26,6 +26,7 @@ import React, {
   useRef,
   useState,
 } from 'react';
+const {default: JSZip} = await import('jszip');
```

Comment:
The async import needs to happen within the download callback itself, otherwise we haven't achieved much as it will always be eagerly loaded 😅

Reply:
I wasnt sure if your components are lazy loaded, but yes i can move it there 👍

Reply:
done